pkg/registry/core/service/storage/storage.go

```diff
@@ -241,6 +235,4 @@ func (r *GenericREST) defaultOnReadService(service *api.Service) error {
 			}
 		}
 	}
-
-	return nil
```

Comment:
I'm relieved to see that this didn't actually drop any error-returning cases

Reply:
Yes, that made this plausible.

Reply:
> I'm relieved to see that this didn't actually drop any error-returning cases

Yep, I had that thought as I started too. :)